app/services/login/user_acceptable_verified_credential_updater_logger.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+module Login
+  class UserAcceptableVerifiedCredentialUpdaterLogger
+    STATSD_KEY_PREFIX = 'user_avc_updater'
+    LOG_MESSAGE = '[UserAcceptableVerifiedCredentialUpdater] - User AVC Updated'
+
+    MHV_TYPE      = 'mhv'
+    DSLOGON_TYPE  = 'dslogon'
+    IDME_TYPE     = 'idme'
+    LOGINGOV_TYPE = 'logingov'
+
+    AVC_TYPE = 'avc'
+    IVC_TYPE = 'ivc'
+
+    def initialize(user_acceptable_verified_credential:)
+      @user_avc = user_acceptable_verified_credential
+    end
+
+    def perform
+      return if user_avc.nil?
+
+      increment_statsd
+      log_info
+    end
+
+    private
+
+    attr_reader :user_avc
+
+    def increment_statsd
+      statsd_keys.each do |key|
+        StatsD.increment(key, 1)
+      end
+    end
+
+    def log_info
+      Rails.logger.info(LOG_MESSAGE, log_payload)
+    end
+
+    def statsd_keys
+      @statsd_keys ||= build_statsd_keys
+    end
+
+    def log_payload
+      @log_payload ||= build_log_payload
+    end
+
+    def added_type
+      @added_type ||= if avc_added?
+                        AVC_TYPE
+                      elsif ivc_added?
+                        IVC_TYPE
+                      end
+    end
+
+    def added_from_type
+      @added_from_type ||= if from_mhv?
+                             MHV_TYPE
+                           elsif from_dslogon?
+                             DSLOGON_TYPE
+                           elsif from_logingov?
+                             LOGINGOV_TYPE
+                           elsif from_idme?
+                             IDME_TYPE
+                           end
+    end
+
+    def build_statsd_keys
+      keys = []
+      return keys unless added_type.present? && added_from_type.present?
+
+      keys << "#{STATSD_KEY_PREFIX}.#{added_from_type}.#{added_type}.added"
+
+      if added_from_type == MHV_TYPE || added_from_type == DSLOGON_TYPE
+        keys << "#{STATSD_KEY_PREFIX}.#{MHV_TYPE}_#{DSLOGON_TYPE}.#{added_type}.added"
+      end
+
+      keys
+    end
+
+    def build_log_payload
+      payload = {}
+      payload[:added_type] = added_type
+      payload[:added_from] = added_from_type if added_from_type.present?
+      payload[:user_account_id] = user_account.id
+      payload[:mhv_uuid] = mhv_credential.mhv_uuid if added_from_type == MHV_TYPE
+      if added_from_type == DSLOGON_TYPE
+        payload[:dslogon_uuid] =
+          dslogon_credential.dslogon_uuid
+      end
+      payload[:idme_uuid] = idme_credential&.idme_uuid
+      payload[:logingov_uuid] = logingov_credential&.logingov_uuid
```

Comment:
For the uuids that we send to the log, it should be:
- mhv_uuid and backing_idme_uuid for mhv
- dslogon_uuid and backing_idme_uuid for dslogon
- idme_uuid for idme
- logingov_uuid for logingov

Reply:
If we passed a `user_verification` into this, we could just take the uuids from the current `user_verification`

Reply:
added `backing_idme_uuid` to the log